app/mailers/ch31_submissions_report_mailer.rb

```diff
@@ -10,7 +10,8 @@ def build(submitted_claims)
 
     opt[:to] =
       if FeatureFlipper.staging_email?
-        Settings.veteran_readiness_and_employment.daily_report.staging_emails.dup
+        # Settings.veteran_readiness_and_employment.daily_report.staging_emails.dup
+        return
```

Comment:
Does this prevent the email from sending all together? Or just return nil. If the latter, why not just delete the `if` block and change to `unless`?

Reply:
Thanks @ericboehs 

This code stems from my suggestion when Amanda and I paired. My thought was to sort of save/comment the staging part in case we ever want to troubleshoot something again. But we can just forget it all together b/c this feature is in prod and all rough stuff should be cleaned up.